src/node_quic_socket.cc

```diff
@@ -987,14 +1050,22 @@ void NewQuicSocket(const FunctionCallbackInfo<Value>& args) {
   CHECK_GE(retry_token_expiration, MIN_RETRYTOKEN_EXPIRATION);
   CHECK_LE(retry_token_expiration, MAX_RETRYTOKEN_EXPIRATION);
 
-  new QuicSocket(
-      env,
-      args.This(),
-      args[0].As<Object>(),
-      retry_token_expiration,
-      max_connections_per_host,
-      options,
-      args[4]->IsTrue() ? QlogMode::kEnabled : QlogMode::kDisabled);
+  uint8_t* sessionResetSecret = nullptr;
```

Comment:
ditto here about the casing :)

That being said … this is read-only data, if I understand correctly, right? In that case it would be helpful to make that more obvious by marking it `const uint8_t*` everywhere and/or using `ArrayBufferViewContents` to read it. The latter allows us to avoid extra allocations for small input, and since the size seems to be fixed at 16, I think using it makes sense.